waspc/run.ps1

```diff
@@ -0,0 +1,38 @@
+# This script defines common commands used during building / developing `waspc` project
+# and makes it easy to run them.
```

Comment:
I would that it duplicates (part of) logic from `run` shell script, and that in the future we plan to merge these two into one, platform-agnostic script, to avoid the duplication.